hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -215,12 +227,25 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 
 	nodePool.Status.NodeCount = int(wantedMachineSet.Status.AvailableReplicas)
 	if !isAutoscalingEnabled {
+		meta.SetStatusCondition(&nodePool.Status.Conditions, metav1.Condition{
+			Type:   hyperv1.AutoscalingEnabledConditionType,
+			Status: metav1.ConditionFalse,
+			Reason: "asExpected",
```

Comment:
pretty sure this should be "AsExpected" for consistency with the rest of the system